UI/cypress/support/step_definitions/results-page.js

```diff
@@ -1,7 +1,11 @@
 import { Given, When, Then, Step } from "@badeball/cypress-cucumber-preprocessor";
 import { kebabCase, KeyStageSubjects } from "../utils";
 
-Given("a user has arrived on the 'Search results' page for {string}", keyStage => {
+Given("a user has arrived on the 'Search results' page for {string}", keyStageSubject => {
```

Comment:
This now matches `a user has arrived on the 'Search results' page for 'Key stage 1'` meaning the results.feature scenarios all fail. I think they need updating to use `Given a user has arrived on the 'Search results' page for 'Key stage 1 English'` or similar and the Given from line 8 in this file can then be deleted.